clients/da-vinci-client/src/main/java/com/linkedin/davinci/consumer/VeniceChangelogConsumerImpl.java

```diff
@@ -182,16 +199,13 @@ public VeniceChangelogConsumerImpl(
 
     }
 
-    LOGGER.info(
-        "Start a change log consumer client for store: {}, with partition count: {} and view class: {} ",
-        storeName,
-        partitionCount,
-        viewClassName);
+    LOGGER.info("Start a change log consumer client for store: {}", storeName);
   }
 
   @Override
   public int getPartitionCount() {
-    return partitionCount;
+    Store store = storeRepository.getStore(storeName);
```

Comment:
Similar here, this will return the original store's partition count in the case of a view store.